addons/website/static/src/js/website.snippets.gallery.js

```diff
@@ -32,16 +35,26 @@ snippets_editor.options.gallery = snippets_editor.Option.extend({
 
         this._temp_mode = this.$el.find("data-mode").data("mode");
         this._temp_col = this.$el.find("data-columns").data("columns");
+        $(document.body).on("media-saved", function() {
+            self.$target.find('.carousel-indicators > li.active').css('background-image', 'url(' + self.$target.find('div.item.active img').attr('src') + ')');
+        });
```

Comment:
Please remove this the code to do that exists in bind_change. But you need to add this missing line:
 $(self.old_media).trigger("saved", [self.active.media, self.old_media]);
in MediaDialog class (website.editor.js line 1850)